qface/idl/listener.py

```diff
@@ -152,6 +152,9 @@ def enterOperationSymbol(self, ctx: TParser.OperationSymbolContext):
         self.operation = Operation(name, self.interface)
         self.parse_annotations(ctx, self.operation)
         self.parse_type(ctx, self.operation.type)
+        modifier = ctx.operationModifierSymbol()
```

Comment:
Move the modifier code before the parse_annotations. So it is in line with the other enter symbol handlers